internal/server/http.go

```diff
@@ -135,5 +138,56 @@ func (s *HTTPServer) CliHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *HTTPServer) SearchHandler(w http.ResponseWriter, request *http.Request) {
-	util.JSONResponse(w, http.StatusOK, map[string]string{"message": "search results"})
+	q := request.URL.Query().Get("q")
+	if q == "" {
+		http.Error(w, "Missing query parameter 'q' ", http.StatusBadRequest)
+		return
+	}
+	if q == "*" {
+		q = ""
+	}
+	f, err := os.Open("https://github.com/DiceDB/playground-web/blob/master/src/data/command.ts")
```

Comment:
This is not correct, let's call `COMMAND` via sdk and fetch list of commands supported for now.